.travis.yml

```diff
@@ -8,20 +8,24 @@ php:
 
 cache:
   directories:
-    - ./vendor
```

Comment:
Ok, so I also pushed the removal of this here because I'm not 110% certain composer really tries to install the correct packages. As you can see in the previous travis run, there's even not any information _what_ packages are installed or not. This makes analyzing this quite hard => https://travis-ci.org/github/fntneves/laravel-transactional-events/jobs/723222584#L221
![image](https://user-images.githubusercontent.com/87493/91911346-2c034600-ecb1-11ea-9188-da7627341d76.png)


It's not travis, but e.g. https://github.com/shivammathur/setup-php explicitly warns about this:
> Please do not cache vendor directory using action/cache as that will have side-effects.

Also now it's nicer as you explicitly see the packages and versions => https://travis-ci.org/github/fntneves/laravel-transactional-events/jobs/723252393#L219
![image](https://user-images.githubusercontent.com/87493/91911316-1857df80-ecb1-11ea-94a6-919ceb1a732e.png)

Reply:
Ok, seems reasonable. Let's keep it disabled!